src/components/PopupWithForm.js

```diff
@@ -1,36 +1,45 @@
-import PopUp from "./PopUp.js";
+import PopUp from "./Popup.js";
 
 export default class PopUpWithForm extends PopUp {
-  constructor(popupSelector, handleFormSubmit) {
-    super({ popupSelector });
-    this._popupForm = this._popupElement.querySelector(".modal__form");
+  constructor(popUpSelector, handleFormSubmit) {
```

Comment:
**Could be improved**

`popupSelector ` was ok because we have word `popup`, so you shouldn't have corrected it to `popUp` everywhere